app/services/metrics/review_turnaround_per_user_project_processor.rb

```diff
@@ -0,0 +1,17 @@
+module Metrics
+  class ReviewTurnaroundPerUserProjectProcessor < MetricProcessor
+    private
+
+    def process
+      review_turnaround_average.each do |review_turnaround|
```

Comment:
how many records will we have here? should we use `find_each` instead? https://apidock.com/rails/ActiveRecord/Batches/find_each